CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-0.0.29 / 2016-11-02
+0.0.30 / TBD
```

Comment:
Just put in the current date please, thanks. Doesn't have to be the day it was merged.